### PR TITLE
Update sauce tunnel keys in Travis config

### PR DESCRIPTION
**Pull request by dfreedm**
Run chrome and firefox in headless configuration

### CODE REVIEW

File: cleanup-passes/travis.ts [hunk 1]
```diff
@@ -124,8 +139,29 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
...
   }
+
+  const sauceCredentials: SauceCredentials =
+      JSON.parse(fs.readFileSync('sauce-credentials.json').toString());
+
+  // Set up sauce keys
+  // remove secure keys
+  const global = te.global.filter((g) => g instanceof String);
+  // encrypt new sauce keys
+  const repoName = `${element.ghRepo.owner.login}/${element.ghRepo.name}`;
```
Comment (usergenic) [inline cleanup-passes/travis.ts:155]: Does this bit work for both orgs and users?

Comment (dfreedm) [reply cleanup-passes/travis.ts:155]: Yeah it should. Running it on `Polymer/polymer` produced the same keys as currently in the config.

Comment (usergenic) [reply cleanup-passes/travis.ts:155]: Ok cool.  The `login` name of the property seemed user-centric and wanted to verify.

**Review by usergenic — APPROVED** [review]

File: cleanup-passes/travis.ts (hunk 1 above)
Comment (dfreedm) [reply cleanup-passes/travis.ts:155]: I'll check real quick to see if there's a more straightforward property.

Comment (dfreedm) [reply cleanup-passes/travis.ts:155]: Nope, `User` only has one property, `login`.

**Pull request merged**